apple/Handlers/RNManualHandler.m

```diff
@@ -19,25 +20,27 @@ - (id)initWithGestureHandler:(RNGestureHandler *)gestureHandler
     _gestureHandler = gestureHandler;
     _shouldSendBeginEvent = YES;
   }
+
   return self;
 }
 
-- (void)touchesBegan:(NSSet<RNGHUITouch *> *)touches withEvent:(UIEvent *)event
+- (void)interactionsBegan:(NSSet *)touches withEvent:(UIEvent *)event
 {
-  [_gestureHandler setCurrentPointerType:event];
```

Comment:
Don't we need this?

Reply:
Correct me if I'm wrong, but it is still present in touchesBegan, just git shows this diff in a weird way

Reply:
Ah, I see. It makes sense to split that between implementations.